src/main/java/g0301_0400/s0350_intersection_of_two_arrays_ii/Solution.java

```diff
@@ -0,0 +1,26 @@
+package g0301_0400.s0350_intersection_of_two_arrays_ii;
+
+// #Easy #Top_Interview_Questions #Array #Hash_Table #Sorting #Binary_Search #Two_Pointers
+
+import java.util.Arrays;
+
+public class Solution {
+    public int[] intersect(int[] nums1, int[] nums2) {
+        Arrays.sort(nums1);
+        Arrays.sort(nums2);
+        int i = 0;
+        int j = 0;
+        int k = 0;
+        while (i < nums1.length && j < nums2.length) {
+            if (nums1[i] < nums2[j]) {
+                i++;
+            } else if (nums1[i] > nums2[j]) {
+                j++;
+            } else {
+                nums1[k++] = nums1[i++];
+                j++;
+            }
+        }
+        return Arrays.copyOfRange(nums1, 0, k);
+    }
```

Comment:
You may keep comments in solutions.

Trailing comment may be moved to separate line.

Reply:
like that ?
![image](https://user-images.githubusercontent.com/93962044/145739824-4495c52d-f899-4ea3-87c3-27590f15c6f6.png)

Reply:
Yes, we may keep comments. We only need to move them, in separate line above.